components/html-block/html-block.js

```diff
@@ -270,11 +273,21 @@ class HtmlBlock extends LitElement {
 		return false;
 	}
 
+	_embedsFeatureEnabled() {
+		return window.D2L?.LP?.Web?.UI?.Flags.Flag('SHIELD-7574-enable-embed-rendering-framework', true);
```

Comment:
This flag doesn't exist yet, but if we're okay with flagging this way, I'll create it.

Reply:
Yeah I'm OK with this as I assume it'll be short-lived anyway.

Reply:
Sounds good. We generally let these sorts of flags soak for ~3 releases before getting rid of them, so it certainly won't be in here for long.